pkg/apis/core/v1beta1/defaults.go

```diff
@@ -51,8 +51,19 @@ func SetDefaults_Seed(obj *Seed) {
 		obj.Spec.Settings = &SeedSettings{}
 	}
 
+	var defaultExcessCapacityReservationConfigs = []SeedSettingExcessCapacityReservationConfig{
+		// This roughly corresponds to a single, moderately large control-plane.
+		{Resources: corev1.ResourceList{corev1.ResourceCPU: resource.MustParse("2"), corev1.ResourceMemory: resource.MustParse("6Gi")}},
+	}
+
 	if obj.Spec.Settings.ExcessCapacityReservation == nil {
-		obj.Spec.Settings.ExcessCapacityReservation = &SeedSettingExcessCapacityReservation{Enabled: true}
+		obj.Spec.Settings.ExcessCapacityReservation = &SeedSettingExcessCapacityReservation{
+			Configs: defaultExcessCapacityReservationConfigs,
```

Comment:
Looking at https://github.com/gardener/gardener/blob/9a5e261b65d1f24a33590cad513f03db72b7d198/pkg/component/seedsystem/seedsystem.go#L111
Don't we need to set `Enabled` to `true` as well during defaulting?

Reply:
I build it like Tim suggested [here](https://github.com/gardener/gardener/pull/8356#discussion_r1299943532). The feature is enabled when `Enabled` is either `true` or `nil`.